TalZivelin/cards2-template.js

```diff
@@ -1,30 +1,51 @@
 
-var cardTools = { // a toolbox object used to group related methods
-//  ^^^^^^^^^could be called anything
 
+
+var cardTools = { 
+  
+ // a toolbox object used to group related methods
+//  ^^^^^^^^^could be called anything
+	
+	isValid: function(num,low,high){
+		if ((typeof num)!="number") //wrong type
+        return NaN;
+		if (num%1 !== 0) //non-integer
+			return NaN;
+		if (num<low || num>high) //out of range
+			return NaN;
+		return true;
+	},
+	
```

Comment:
Its good practice to have an alert along with the return of an error or false. This helps you for coding along the way.